handbook/okrs/CY22-Q2/team.md

```diff
@@ -7,7 +7,7 @@ _Objective_: Design, implement, maintain, and enhance a modern data platform tha
 _Key Result_: At least one new end-to-end use case is operationalized on the data platform, including data capture, modeling, transformation, orchestration, and discovery. [1/5]
 
 #### O1KR2 
-All 3 codebases and 4 cloud infrastructure services that currently comprise the data platform are tracked via source control, and are deployed via CI/CD. [7/14]
+All 3 codebases and 4 cloud infrastructure services that currently comprise the data platform are tracked via source control, and are deployed via CI/CD. [13/14]
```

Comment:
The denominator is a little misleading here: originally we were tracking 3 codebases (Vestapol, inq-data-resources, and inquirer-dbt) and 4 Google Cloud services (Cloud Storage, BigQuery, Cloud Composer, Cloud Functions) for both source control and deployment, with `(3 + 4) * 2 = 14`. Of those 7 codebases and services, Cloud Functions have not been moved to CI/CD, nor are they tracked in Terraform.

As we grown more familiar with Terraform, we realize that there is much more infrastructure related to these services that we would also want to manage in Terraform (e.g. IAM bindings, service accounts)